README.md

```diff
@@ -203,3 +203,27 @@ If you would like to add your organization to the list, please comment on our
 [slack]: https://cloud-native.slack.com/archives/CGG7NFUJ3
 [slack-join]: https://slack.cncf.io
 [slack-img]: https://img.shields.io/badge/slack-join%20chat%20%E2%86%92-brightgreen?logo=slack
+
+## Releases
```

Comment:
This doesn't feel like the right place, needs to be much higher in the doc. Also, rather than being under "releases" heading, perhaps it should be phrased more in terms of compatibility guarantees.

Lastly, as a general observation, we don't have a clear separation of responsibilities between readme and the docs. We should think about it.

Reply:
> Would it make sense to move this before: "Related Repositories"?

+1

> What are the use cases for README vs. docs?

Don't know, it varies widely between projects. For example, [k8s readme][1] is very short, doesn't even have such sections as license & contributing (probably expecting people to look into respective files).

> It seems the docs cater for operational use cases

Not necessarily, the Intro & Features pages in the docs largely repeat what's in the README. I think it would be good to move more stuff into docs and keep the README more development-focused, to minimize duplication of content. Another useful template is [Envoy][2], it's README is slightly longer than k8s's, but also avoids listing features and such.

But that doesn't actually answer the question where to put compatibility guarantees, I can still see both places as candidates.

[1]: https://github.com/kubernetes/kubernetes
[2]: https://github.com/envoyproxy/envoy

Reply:
> But that doesn't actually answer the question where to put compatibility guarantees, I can still see both places as candidates.

I've added this topic for further discussion in the upcoming Jaeger community call to see if the group has a particular preference.